### PR TITLE
Remove semicolons from strings

### LINKED ISSUE #16275: [Bug]: netdata-go not found: create zip: modules/prometheus/integrations/cs:go.md: malformed file path "modules/prometheus/integrations/cs:go.md": invalid char ':'
### Bug description

In FreeBSD we use the following link to allow for an easy port installation. Going from v0.56.0 to v0.56.1 it stopped working.
The following error is displayed when trying to fetch from this location: 

> not found: create zip: modules/prometheus/integrations/cs:go.md: malformed file path "modules/prometheus/integrations/cs:go.md": invalid char ':'

https://[proxy.golang.org/github.com/netdata/go.d.plugin/@v/v0.56.1.mod](https://proxy.golang.org/github.com/netdata/go.d.plugin/@v/v0.56.1.mod)

https://[proxy.golang.org/github.com/netdata/go.d.plugin/@v/v0.56.0.mod](https://proxy.golang.org/github.com/netdata/go.d.plugin/@v/v0.56.0.mod)

### Expected behavior

The fetch link works as expect; and higher verisons:
https://[proxy.golang.org/github.com/netdata/go.d.plugin/@v/v0.56.1.mod](https://proxy.golang.org/github.com/netdata/go.d.plugin/@v/v0.56.1.mod)

### Steps to reproduce

Open in webbrowser and display error message

### Installation method

from source

### System info

```shell
[/usr/ports/net-mgmt/netdata-go] git:(main*)$ uname -a
FreeBSD freebsd.vm 15.0-CURRENT FreeBSD 15.0-CURRENT #0 main-n265954-fc9f1d2c6391: Sun Oct 15 18:05:08 CEST 2023     dries@freebsd.vm:/usr/obj/usr/src/amd64.amd64/sys/GENERIC amd64

[/usr/ports/net-mgmt/netdata-go] git:(main*)$ uname -K
1500001
```

### Netdata build info

```shell
Irrelevant, this bug is with fetching sources.
```

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

Edge case where semicolon was inside a string that was used as filename.

Closes #16275 

##### Test Plan

Tested on both repos, only the affected file changed, and got fixed.

### CODE REVIEW

**Review by ilyam8 — DISMISSED** [review]

File: integrations/gen_docs_integrations.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ def clean_string(string):
     The string represents an integration name, as it would be displayed in the final text
     """
 
-    return string.lower().replace(" ", "_").replace("/", "-").replace("(", "").replace(")", "")
+    return string.lower().replace(" ", "_").replace("/", "-").replace("(", "").replace(")", "").replace(":","")
```
Comment (ilyam8) [inline integrations/gen_docs_integrations.py:100]:
```suggestion
    return string.lower().replace(" ", "_").replace("/", "-").replace("(", "").replace(")", "").replace(":", "")
```

**Review by ilyam8 — APPROVED** [review]

**Review by underhood — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]
